scripts/dev/src2asciidoc.py

```diff
@@ -512,6 +513,7 @@ def regenerate_manpage(filename):
             if action_data is not None:
                 groupdata.append(action_data)
         groups.append('\n'.join(groupdata))
+    # pylint: enable=protected-access
```

Comment:
I think you can move the disable/enable around the `for group in parser._action_groups:` line here.